sql/hive/src/main/scala/org/apache/spark/sql/hive/client/ClientInterface.scala

```diff
@@ -109,6 +109,9 @@ private[hive] trait ClientInterface {
   /** Returns the name of the active database. */
   def currentDatabase: String
```

Comment:
Yes we should - the only thing is that the simple catalog in non-Hive doesn't support databases yet.